README.md

```diff
@@ -51,4 +51,5 @@ Click outside of textboxes before placing or removing barrels.
 
 **Aryamanarora** - Metadata
 
-**Ericw31415** - Tidying up & improving info stuff
+# (Note: ericw31415 doesn't like having his username capitalized.)
+**ericw31415** - Tidying up & improving info stuff
```

Comment:
That was the original idea, although the current method is a bit simpler simply because I lack the experience to actively judge usefulness. Not to put down the metadata changes, but it's a good example: I have no idea what adding metadata actually does for us. I know it helps search engines, but that to me seems exceedingly minor. Internal cleanup or feature additions on the other hand seem so very much more.

But for all I know, an experienced webdev would call that metadata out as the single greatest thing.